File: Data Structures & Algorithms/sort-an-array/submission-0.py
class Solution:
    def sortArray(self, nums: List[int]) -> List[int]:
        n=len(nums)
        for i in range(0,n):
          j=i+1
          while(j<n):
            if(nums[i]>nums[j]):
              nums[i],nums[j]=nums[j],nums[i]
              j+=1
            elif(nums[i]<nums[j]):
              j+=1
            else:
              j+=1
        return nums
            
            

File: Data Structures & Algorithms/sort-colors/submission-0.java
class Solution {
    public void sortColors(int[] nums) {
      int n=nums.length;
      for(int i=0;i<n;i++){
        int j=i+1;
        while(j<n){
          if(nums[i]>nums[j]){
            int temp=nums[i];
            nums[i]=nums[j];
            nums[j]=temp;
            j++;
          }
          else if(nums[i]<nums[j]){
            j++;
          }
          else{
            j++;
          }
        }
      }
      for(int i=0;i<n;i++){
      System.out.println(nums[i]);
      }  
    }
}